README.md

```diff
@@ -76,70 +76,81 @@ This module have the following API:
 
 Supported arguments are:
 
- - `options` - Object - A Google Cloud Storage [auth object][gcs-auth] - Required.
- - `bucket` - String - Name of the bucket in Google Cloud Storage to write and read files to/from - Required.
- - `metadata` - Object - A bucket [metadata object][gcs-bucket-meta] - Optional.
+* `options` - Object - A Google Cloud Storage [auth object][gcs-auth] -
+  Required.
+* `bucket` - String - Name of the bucket in Google Cloud Storage to write and
+  read files to/from - Required.
+* `metadata` - Object - A bucket [metadata object][gcs-bucket-meta] - Optional.
 
 Events:
 
- - `storage info` - General info on the underlaying storage. Emits with: `message` and `metaObject`.
+* `storage info` - General info on the underlaying storage. Emits with:
+  `message` and `metaObject`.
+
+### get(fileName)
```

Comment:
Maybe should indicate return type:
What about just writing types or some such??

###` get(filename: string): Promise<string>`